kotlin/feature-layer-selection/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.esri.arcgisruntime.sample.featurelayerselection.MainActivity">
+
+    <com.esri.arcgisruntime.mapping.view.MapView
```

Comment:
As mentioned above, it's possible to remove the constraint layout and just have the mapview here